src/coreclr/nativeaot/System.Private.Reflection.Execution/src/Internal/Reflection/Execution/TypeLoader/TypeCast.cs

```diff
@@ -21,6 +22,8 @@ namespace Internal.Reflection.Execution
     /////////////////////////////////////////////////////////////////////////////////////////////////////
 
     // This is not a general purpose type comparison facility. It is limited to what constraint validation needs.
+    [StackTraceHidden]
```

Comment:
What's the purpose of this change?

Reply:
I've added it to the wrong TypeCast at first, it's already reverted. (I've added it to the real one so it's the same as the CoreCLR one)